package.json

```diff
@@ -84,13 +84,13 @@
     "rollup": "^3.20.2",
     "rollup-plugin-dts": "^5.3.0",
     "rollup-plugin-postcss": "^4.0.1",
-    "rollup-plugin-typescript2": "^0.30.0",
+    "rollup-plugin-typescript2": "^0.35.0",
```

Comment:
Not sure how version bumps are handled in this package, but they're probably not needed for this feature. I'd revert this as well.

Reply:
Woops - yeah, this was accidental.